aten/src/ATen/fbgemm_utils.h

```diff
@@ -0,0 +1,21 @@
+#pragma once
```

Comment:
Should this file be under quantized/cpu?

Reply:
This "fbgemm_utils.h" might be reused by some routines in other places, so it is put in the same directory as "cpp_custom_type_hack.h". In addition, we need to change the TARGETS and CMakefile in order to enable the header files in "quantized/cpu" location.